src/main/java/com/codessquad/qna/web/UserController.java

```diff
@@ -46,28 +48,70 @@ public String getList(Model model) {
     }
 
     @GetMapping("/{id}")
-    public String getOneUserProfile(@PathVariable long id, Model model) {
-        model.addAttribute("user", userRepository.findById(id).orElseThrow(NoUserException::new));
+    public String getOneUserProfile(@PathVariable long id, Model model, HttpSession session) {
```

Comment:
```suggestion
    public String userProfile(@PathVariable long id, Model model, HttpSession session) {
```

이렇게 바꾸는 편이 훨씬 잘 읽히지 않을까요?